lib/code-caller/code-caller-container.js

```diff
@@ -401,6 +403,8 @@ class CodeCallerContainer {
     this.stderrStream = new MemoryStream();
     this.stdinStream.pipe(stream);
     this.container.modem.demuxStream(stream, this.stdoutStream, this.stderrStream);
+    this.stdoutStream.setEncoding('utf8');
+    this.stderrStream.setEncoding('utf8');
```

Comment:
This was necessary to ensure we get strings in the handlers, which is consistent with how the native code caller works.